markdown/admin/ambari-admin.html.md.erb

```diff
@@ -368,6 +348,49 @@ All of the listed steps are mandatory. This ensures that HAWQ service remains fu
     4. Restart HAWQ service to propagate the configuration change to all Ambari agents.
 
     This will synchronize the password on the host machines with the password that you specified in Ambari.
+    
+## <a id="manual-cfg-norestart"></a>Updating HAWQ Configuration Without Cluster Restart
+There may be circumstances, such as during dynamic cluster expansion, when you must update HAWQ configuration parameters but cannot tolerate cluster downtime in your Ambari-managed cluster. In these situations, perform the following workaround to update and reload HAWQ configuration changes without a complete cluster restart.
+
+**Note**: Use this procedure for setting or updating only those HAWQ server configuration parameters with a set classification of reload.
+
+1.  Use the Ambari console UI to set and save the desired HAWQ configuration parameter(s) via the **HAWQ** service **Settings**, **Configs >Settings**, and/or **Configs >Advanced** tabs. Make sure to note the configuration parameter names and new values. You may need to hover the mouse over the configuration field to identify the HAWQ parameter name.
+    
+    **Note**: After updating the configuration parameters, do *not* restart the HAWQ service. 
+
+1.  Use HAWQ command line utilities to update the same HAWQ configuration parameters:
+    1. Log in to the HAWQ master host as a HAWQ administrator and source greenplum_path.sh:
+
+        ``` shell
+        $ ssh gpadmin@<master>
+        gpadmin@master$ source /usr/local/hawq/greenplum_path.sh
+        ```
+        
+    2. Identify the current value of the HAWQ configuration parameter:
+
+        ``` shell
+        gpadmin@master$ hawq config -s <parameter_name>
+        ```
+        
+    3. Set the configuration parameter to the new value:
+
+        ``` shell
+        gpadmin@master$ hawq config -c <parameter_name> -v <new_value>
+        ```
+        
+        Perform Steps 2 and 3 for each configuration parameter you set or updated via Ambari.
+        
+3. Reload the HAWQ configuration; this operation does not restart the cluster:
+
+    ``` shell
+    gpadmin@master$ hawq stop cluster --reload
+    ```
+    
+4. Verify that each parameter was updated:
+
+    ```shell
+    gpadmin@master$ hawq config -s <parameter_name>
+    ```
```

Comment:
I guess I was misunderstanding something from the earlier workaround:  "When the HAWQ service is ready to be restarted via Ambari, Ambari will refresh the new configurations."  I guess I'm not entirely sure what that means either.  So when exactly is the change "complete"?